### DISCUSSION TITLE
Error message trying to upgrade to Firefly 1.4

### DISCUSSION THREAD

**Discussion by clearspine** [iotaledger/firefly#2748] [Help & Support] [answered]
App Version: 1.3.3

platform: macOS
platformVersion: 10.14
platformArchitecture: x64
cpuCount: 4
totalMem: 8192.0 MB
freeMem: 114.9 MB

Error Type: Main Unhandled Promise Rejection

Error: Uncaught Error: Cannot download "https://dl.firefly.iota.org/firefly-desktop-1.4.0.zip", status 403: 

Is anyone else having this problem?  Any thoughts as to what I should do?

**Comment by MiKeF05** [comment, marked as answer]
The latest release 1.4 is currently not available. There is an issue with the profile history. So you need to wait until a new version will be released.

**Comment by clearspine** [comment]
Interesting, because on Firefly it says that 1.3.3 is out of date, and invites me to update to 1.4


-----Original Message-----

**Reply by MiKeF05** [reply]
Yeah the new release was made but the files were temporarily disabled

**Comment by clearspine** [comment]
Thanks very much for your help.  I'll just wait for the release


-----Original Message-----
